Index: src/main/java/domian/Employee.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/domian/Employee.java	(date 1539519400000)
+++ src/main/java/domian/Employee.java	(date 1539519711000)
@@ -11,6 +11,8 @@
     private String lastname;
     private  double salary;
     private  double salary2;
+    private  double salary2;
+
 
 
     @Id
